jenkins-scripts/ign_rendering-default-devel-windows-amd64.bat

```diff
@@ -5,7 +5,7 @@ set VCS_DIRECTORY=ign-rendering
 set PLATFORM_TO_BUILD=x86_amd64
 set IGN_CLEAN_WORKSPACE=true
 :: ogre2 from osrf vcpkg-ports
-set DEPEN_PKGS="dlfcn-win32 cuda freeimage ogre ogre2 gts glib"
+set DEPEN_PKGS="boost dlfcn-win32 cuda freeimage ogre ogre2 gts glib"

```

Comment:
All of boost installed (predictably) a lot. I'm not sure if there's a specific set of boost libraries that we ought to add instead.

Reply:
> All of boost installed (predictably) a lot. I'm not sure if there's a specific set of boost libraries that we ought to add instead.

There are a good number of boost packages in vcpkg https://github.com/microsoft/vcpkg/tree/master/ports . We will need to investigate where the boost message came from and what is really looking for (maybe boost-core is enouhg).

Reply:
boost is being used by ign-rendering since there are parts of Ogre that needs it. I change the PR to use boost-core, let's see what happened: [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign_rendering-pr-win&build=1696)](https://build.osrfoundation.org/job/ign_rendering-pr-win/1696/)